mlir/test/Integration/Dialect/LLVMIR/CPU/lit.local.cfg

```diff
@@ -18,5 +35,12 @@ if config.riscv_vector_emulator_executable:
         emulation_cmd = emulation_cmd + " " + config.riscv_vector_emulator_options
     emulation_cmd = emulation_cmd + " " + lli_cmd + " --march=riscv64 -mattr=+v "
     config.substitutions.append(("%lli", emulation_cmd))
+elif config.arm_emulator_executable:
+    # Run test in qemu emulator.
+    emulation_cmd = config.arm_emulator_executable
+    if config.arm_emulator_options:
+        emulation_cmd = emulation_cmd + " " + config.arm_emulator_options
+    emulation_cmd = emulation_cmd + " " + lli_cmd + " --march=aarch64 -mattr=+neon "
```

Comment:
Why just `-mattr=+neon` here?

Reply:
also not clear to me what the rationale here is. I don't believe `arm_emulator_executable` was intended to mean run all the integration tests under emulation, it was intended to support running the integration tests for ISAs like SVE and SME enabled under
- MLIR_RUN_ARM_SVE_TESTS
- MLIR_RUN_ARM_SME_TESTS

where hardware isn't yet widely available. I think we need to mindful here because this will mean these tests are unnecessarily run under emulation on AArch64 hosts and will take longer to run.

Reply:
Reverted changes to test/Integration/Dialect/LLVMIR/CPU/lit.local.cfg and sinked substitutions rule of `%mlir_native_utils_lib_dir` to test/Integration/Dialect/SparseTensor/CPU/lit.local.cfg.